lib/components/Body.js

```diff
@@ -50,6 +56,15 @@ var Body = React.createClass({
     if (isDate(body)) {
       return $(Data, null, body.toString());
     }
+    if (isSet(body)) {
+      var contents = Array.from(body)
+        .map(function(item) {
```

Comment:
I think it's worth extracting the `value -> string` function out of the component, so it can then be recursive.